src/NeoPixelBus.h

```diff
@@ -36,6 +36,7 @@ License along with NeoPixel.  If not, see
 
 #include "internal/NeoSettings.h"
 
+#include "internal/MonoColor.h"
```

Comment:
While I like MonoColor as the class name/filename, it doesn't perfectly match the other color names (RgbColor, RgbwColor).  The alternative would be WColor, but I really don't like that color name.
So don't consider this comment that it needs to change, just consider some other names and we are capturing that here that we did and why we stuck with MonoColor.